### PR TITLE
Reinstate bouncy spongie

### PR DESCRIPTION
**Pull request by jamierocks**
This was removed a while back, for not looking all too professional. However I think it adds some extra flare to the site and improves its overall look.

I also feel that it is fitting with the Sponge brand, with its playful aspects - something I would imagine would be good to promote, when the Sponge project literally revolves around a game.

Well thats my 2 cents anyway...

### CODE REVIEW

**Comment by jamierocks** [comment]
so i be that very cool spongie auto bot thingy.

here ya go -> http://dev.jamiemansfield.me/spongehome/

**Comment by Spongy** [comment]
A preview for this pull request is available at https://spongy.github.io/SpongeHome-PRs/129/.

Here are some links to the pages that were modified:

- modified: https://spongy.github.io/SpongeHome-PRs/129/

File: src/scss/_index_sections.scss [hunk 1]
```diff
@@ -158,6 +158,104 @@
...
+    }
+    75% {
+      transform: translate(0, -15px) rotate(0deg);
+    }
+    100% {
+      transform: translate(0, 0) rotate(-7deg);
+    }
+  }
+
+  @-webkit-keyframes spongieDance {
```
Comment (stephan-gh) [inline src/scss/_index_sections.scss:180]: This is not needed, [autoprefixer](https://github.com/postcss/autoprefixer) will automatically handle adding the necessary prefixes.

File: src/scss/_index_sections.scss [hunk 2]
```diff
@@ -158,6 +158,104 @@
...
+      -ms-transform: translate(0, -15px) rotate(0deg);
+    }
+    100% {
+      -ms-transform: translate(0, 0) rotate(-7deg);
+    }
+  }
+
+  .spongie-dance {
+    animation: spongieDance ease-in 3s infinite;
+    -webkit-animation: spongieDance ease-in-out 3s infinite;
```
Comment (stephan-gh) [inline src/scss/_index_sections.scss:254]: As above

**Review by stephan-gh — CHANGES_REQUESTED** [review]

**Comment by progwml6** [comment]
This was intentionally removed in the past. It can not go back in unless it is agreed upon by staff.

**Comment by Zidane** [comment]
I understand the OP's sentiments but this won't be returning.

**Pull request closed without merging**